_includes/sidebar-data-v19.2.json

```diff
@@ -43,7 +43,7 @@
           {
             "title": "Learn CockroachDB SQL",
             "urls": [
-              "/${VERSION}/learn-cockroachdb-sql.html"
+              "/${VERSION}/learn-cockroachdb-sql-movr.html"
```

Comment:
We should not do this IMO for several reasons: (1) we throw away all our PageRank on the existing page built up over several years, (2) the existing page becomes an orphan since AFAICT it's still in the site.  We should instead update the existing "Learn CRDB SQL" page with the MovR content IMO

Reply:
Both totally legitimate reasons to just update the file with the same name rather than create an additional file and have two separate files! I'll fix this.